test/functional/partners/providers_controller_test.rb

```diff
@@ -23,6 +22,10 @@ def provider_params
     {subdomain: 'troloro', org_name: 'foo-org', email: 'foo@example.net', first_name: 'Tyler', last_name: 'Durden', api_key: @partner.api_key, open_id: "openid"}
   end
 
+  test 'routes' do
+    assert_routing({ action: 'post', path: "http://#{master_account.domain}/partners/providers" }, { action: 'index', format: 'json', controller: 'partners/providers' })
```

Comment:
This assertion now passes with `action: 'index'` only.. any idea why?

Reply:
according to docs it should be `method` unless I'm missing somthing 
https://apidock.com/rails/ActionDispatch/Assertions/RoutingAssertions/assert_routing